documentation/ame-guide/modules/ROOT/pages/edit-models.adoc

```diff
@@ -20,20 +23,19 @@ image::ame-ui-overview-workspace-numbers.png[width=100%]
 +
 For more information, see xref::namespaces-references.adoc#namespaces-external-references[Namespaces and external references].
 
-2. The Workspace sidebar also offers all available default elements for your Aspect Models. From here you can add new Properties, Characteristics, Entities, Constraints, Operations, Events, and Traits by dragging an element from the sidebar to the graph area canvas.
+2. The Workspace sidebar also offers all available default elements for your Aspect Models.
```

Comment:
The Workspace sidebar also offers all types of elements to drag and drop into your Aspect Model.